mypy.ini

```diff
@@ -1,5 +1,5 @@
 [mypy]
-ignore_missing_imports = True
+strict = True
```

Comment:
one-size-fits-all seems to me to be a strange goal for this configuration.  Different projects will have different imports: some of which will have complete type stubs, some of which will have partial type stubs, some of which will have none.

I would recommend that you go strict everywhere, and ignore missing imports explicitly for specific packages as needed, rather than unconditionally ignore-missing-imports.  That makes it clearer what is going on and - as here, perhaps! - encourages contributions to dependencies that are nearly well-typed.

However I do not care enough to open a pull request so I will leave you to it.

Reply:
> I would recommend that you go strict everywhere, and ignore missing imports explicitly for specific packages as needed, rather than unconditionally ignore-missing-imports. That makes it clearer what is going on and - as here, perhaps! - encourages contributions to dependencies that are nearly well-typed.

See also https://github.com/jaraco/skeleton/issues/98#issuecomment-1812838723 ;)